module/mempool/consensus/exec_fork_suppressor.go

```diff
@@ -132,15 +117,6 @@ func (s *ExecForkSuppressor) Add(newSeal *flow.IncorporatedResultSeal) (bool, er
 		return false, nil
 	}
 
-	// STEP 4: check whether wrapped mempool ejected the newSeal right away;
```

Comment:
I've simplified this checks since we don't support this scenario anymore + it doesn't make sense when underlying mempool has extra logic on querying(enforcing multiple receipts)